scripts/install/install_macos.sh

```diff
@@ -272,6 +278,36 @@ check_prereqs()
   decrease_indent
 }
 
+# Test connection to BindPlane if it was specified
+connection_check()
+{
+  info "Help Me Obi-Wan Kenobi!"
```

Comment:
Leftover debug statement?

Reply:
Yessir, good catch. Thanks.

Reply:
On this, I've noticed an inconsistency between OSX and unix scripts, where we run some checks before parsing arguments on OSX, but not in the unix script. Should that be fixed?

Reply:
We should make them consistent, but I'm not overly concerned about it, certainly wouldn't be against a follow-up PR fixing that 😄